modules/kubernetes-addons/variables.tf

```diff
@@ -220,6 +224,23 @@ variable "aws_for_fluentbit_irsa_policies" {
   default     = []
 }
 
+variable "aws_for_fluentbit_cw_log_group_name" {
+  type        = string
+  description = "FluentBit CloudWatch Log group name"
+  default     = null
```

Comment:
lets provide a default?

Reply:
We have an option in locals to create a opinionated log group if the users doesn't provide any

`log_group_name       = var.cw_log_group_name == null ? "/${var.eks_cluster_id}/worker-fluentbit-logs" : var.cw_log_group_name`